revealer_bot/network_status.py

```diff
@@ -11,9 +11,7 @@ async def get_lynx_network_status(interaction):
     thread = await initial_reply.create_thread(name=thread_name, auto_archive_duration=60)
 
     # Fire off a request to the network status endpoint, along with a message to the user that info is incoming.
-    status_response = requests.get(
-        "https://lynx-1.nucypher.network:9151/status/?json=true", verify=False
-    )
+    status_response = requests.get("https://lynx-1.nucypher.network:9151/status/?json=true", verify=False)
```

Comment:
Just formatting changes stemming from increasing max-line-length to 160 characters